subsys/bluetooth/audio/aics_internal.h

```diff
@@ -104,6 +104,7 @@ struct bt_aics_server {
 
 /* Struct used as a common type for the api */
 struct bt_aics {
+	bool client_instance;
```

Comment:
Same concern here as in MICS about the size difference between client and service instance.

Reply:
See https://github.com/zephyrproject-rtos/zephyr/pull/35823#discussion_r657012312